model/perf.go

```diff
@@ -86,6 +90,7 @@ func (result *PerformanceResult) Find() error {
 	} else if err != nil {
 		return errors.Wrap(err, "problem finding result config")
 	}
+	result.Setup(env)
```

Comment:
this can't possibly do what we mean? probably omit.

Reply:
nope, it is intentional

Reply:
`AttachResultData`, `AttachAuxilaryData`, and `AttachRollups` all fail without restoring `env`.